site/src/_includes/layouts/head.njk

```diff
@@ -1,10 +1,10 @@
 <head>
   <meta charset="utf-8">
   <title>{{ [title, "|", site.title] | join(" ") }}</title>
-  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="description" content="{{ description }}">
+  {% set page_url = [site.url, page.url | url] | join -%}
   <meta name="twitter:card" content="summary">
-  <meta name="twitter:site" content="https://getquick.link">
```

Comment:
@addyosmani I removed this because it's supposed to be a Twitter handler. Is there a Twitter account for Quicklink? If not, I guess the removal is OK.